nx-X11/programs/Xserver/hw/nxagent/Window.c

```diff
@@ -466,6 +466,18 @@ FIXME: Do all the windows for which nxagentWindowTopLevel(pWin)
     nxagentRedirectWindow(pWin);
   }
 
+  if ((nxagentRealWindowProp) && (!nxagentWindowTopLevel(pWin)))
+  {
+    Atom prop = MakeAtom("NX_REAL_WINDOW", strlen("NX_REAL_WINDOW"), True);
```

Comment:
> Maybe add that to the nxagentAtomNames array in Atoms.c

Those nxagentAtomNames are defined as static and only used inside Atoms.c.  They get used in nxagentQueryAtoms. But we don't need this.